pkg/subscriber/helmrepo/helm_subscriber_item.go

```diff
@@ -288,27 +288,26 @@ func (hrsi *SubscriberItem) manageHelmCR(indexFile *repo.IndexFile, repoURL stri
 			return err
 		}
 
+		dplkey := types.NamespacedName{
+			Name:      dpl.Name,
+			Namespace: dpl.Namespace,
+		}
+		pkgMap[dplkey.Name] = true
```

Comment:
placing this `= true` here is it ok? Or can it be in somewhere more logically accurate.

Reply:
`pkgMap` is more like a placeholder, it not used at all..

Since we don't want to change functions interfaces... I kept the `pkgMap`